### PR TITLE
Resolving Code Conflicts & remove Unnecessary Code

### PR DESCRIPTION
**Pull request by ImYenil**
Resolving Code Conflicts & remove Unnecessary Code

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java [hunk 1]
```diff
@@ -20,11 +20,11 @@
 
 package com.tomkeuper.bedwars.arena;
 
+import com.google.common.io.ByteArrayDataOutput;
+import com.google.common.io.ByteStreams;
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:24]: What is this for?

Comment (ImYenil) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:24]: > What is this for?

image

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java [hunk 2]
```diff
@@ -33,8 +33,6 @@
 import com.tomkeuper.bedwars.configuration.Sounds;
 import com.tomkeuper.bedwars.stats.PlayerStats;
 import com.tomkeuper.bedwars.support.papi.SupportPAPI;
-import com.google.common.io.ByteArrayDataOutput;
-import com.google.common.io.ByteStreams;
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:37]: Aahh you removed it here and then re-added it at the top

Comment (ImYenil) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:37]: > Aahh you removed it here and then re-added it at the top

Looks like I used the Optimize Imports feature

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java [hunk 3]
```diff
@@ -392,18 +390,6 @@ public static boolean isBuildProtected(Location l, IArena a) {
...
-                if (o.getLocation().distance(l) <= a.getConfig().getInt(ConfigPath.ARENA_GENERATOR_PROTECTION)) {
-                    return true;
-                }
-            }
-        }
-        for (IGenerator o : a.getOreGenerators()) {
-            if (o.getLocation().distance(l) <= a.getConfig().getInt(ConfigPath.ARENA_GENERATOR_PROTECTION)) {
-                return true;
-            }
-        }
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:406]: Why did you remove the generator protections?

Comment (ImYenil) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:406]: > Why did you remove the generator protections?

Check out the OreGenerator class. Added again.

Comment (tomkeuper) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Misc.java:406]: Ahh so it was double

**Comment by tomkeuper** [comment]
Did you double check if it still works with ore gens?

**Comment by ImYenil** [comment]
> Did you double check if it still works with ore gens?

Yes i checked

**Review by tomkeuper — APPROVED** [review]

**Review by porudev — APPROVED** [review]
